docs/upgrade/index.md

```diff
@@ -0,0 +1,35 @@
+---
+html_meta:
+  "description": "Plone 6 upgrade Guide"
+  "property=og:description": "Plone 6 upgrade guide"
+  "property=og:title": "Plone 6 upgrade guide"
+  "keywords": "Plone, upgrade"
+---
+
+(upgrade-guide-label)=
+
+# Plone 6 upgrade guide
```

Comment:
Not topic of this PR. I just thought if the navi on the left side could be freed of repeated mentioning of 'Plone 6'.
And keeping the html_meta section to have 'Plone6' mentioned in Google results.
![image](https://user-images.githubusercontent.com/1144737/179532093-55a76264-fcf5-4d27-b292-a6bab9efdec8.png)

Reply:
I created a new issue to handle that in https://github.com/plone/documentation/issues/1272. It crossed my mind, too.